ftgo-order-history-service/src/main/java/net/chrisrichardson/ftgo/cqrs/orderhistory/messaging/OrderHistoryServiceMessagingConfiguration.java

```diff
@@ -11,7 +12,7 @@
 import org.springframework.context.annotation.Import;
 
 @Configuration
-@Import({CommonConfiguration.class, TramNoopDuplicateMessageDetectorConfiguration.class, TramEventSubscriberConfiguration.class})
+@Import({CommonConfiguration.class, TramNoopDuplicateMessageDetectorConfiguration.class, TramEventSubscriberConfiguration.class, KafkaConsumerFactoryConfiguration.class})
```

Comment:
Requiring an application to `@Import` `KafkaConsumerFactoryConfiguration` doesn't seem right. This is a low-level detail that should be hidden.

I believe that upgrading to eventuateTramVersion=0.25.0.RELEASE will fix this